opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -178,15 +179,22 @@
             "metadata": {
                 "description": "HealthCheck cache interval value in seconds."
             }
+        },
+        "customTShirtSize": {
+            "type": "object",
+            "defaultValue": {},
+            "metadata": {
+                "description": "Define an object for a custom tshirt size of the deployment. WARNING: Intended for experimenting or debugging use with care."
+            }
         }
     },
     "variables": {
         "dnsNamePrefix": "[toLower(substring(concat('opencga', uniqueString(subscription().subscriptionId,parameters('rgPrefix'))), 0, 16))]",
         "sizingOptions": [
             {
                 "type": "0 = Small",
-                "avereEnabled": false,
-                "azureFilesEnabled": true,
+                "avereEnabled": true,

```

Comment:
why are we enabling avere on size 0?

Reply:
Nope this is the kind of mistake I was trying to avoid with this change! I was testing with avere by editing the default size. Will roll back.